site/includes/sign-on-off.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+	<div class="row">
+{{#is signed "off"}}
```

Comment:
That may not always be desirable or possible so my use case is still valid and implementers have been asking for just this for a long time. Seen the screenshot below.